src/frontend/services/fundraiser.js

```diff
@@ -0,0 +1,30 @@
+import {
+  getApiUrl,
+  getApplicationJsonHeader,
+  getAuthHeader,
+  handleResponse,
+} from "../utlis";
+
+export const createFundraiser = (data) => {
+  return fetch(getApiUrl("fundraiser"), {
+    method: "POST",
+    headers: {
+      ...getApplicationJsonHeader(),
+      ...getAuthHeader(),
+    },
+    body: JSON.stringify({data}),
+  })
+    .then(handleResponse)
+    .then((data) => data.data);
+};
+
+export const getFundraiserById = (id) => {
```

Comment:
we would need a getFundraiserByUserId as well to show it in myFundraiser page,

Reply:
yes , will create it eventually